examples/auth-generate-token/page.md

```diff
@@ -0,0 +1,151 @@
+# Generate a JWT token to authenticate clients
+
+Use JWT tokens to securely authenticate users within your application.
+
+JWT tokens enable you to grant users access to Ably's real-time messaging services. They are most commonly used to issue time-limited credentials that securely authorize users to connect to Ably channels.
+
+JWT tokens are an essential component of applications that require secure and scalable real-time communication. Before any application can use any of the Ably products whether they be Pub/Sub, Spaces, Chat, or LiveSync, they'll need to first authenticate.
```

Comment:
```suggestion
The first step for an application to communicate with Ably requires them authenticating.
```

I think we should replace the deleted info here with explaining about authUrl and token renewal. WDYT?